az-core/src/main/java/azkaban/Constants.java

```diff
@@ -226,6 +226,20 @@
     public static final String QUEUEPROCESSING_ENABLED = "azkaban.queueprocessing.enabled";
 
     public static final String SESSION_TIME_TO_LIVE = "session.time.to.live";
+
+    // allowed max size of project dir in mb
+    public static final String PROJECT_DIR_MAX_SIZE = "azkaban.project_cache_max_size_in_mb";
```

Comment:
Why does this change show up here? This seems to belong to another change.

Reply:
@HappyRay there was a merge conflict while I was updating my branch to master's head. Would it be better to close this PR and open a new one?